fsd_utils/authentication/decorators.py

```diff
@@ -0,0 +1,65 @@
+from functools import wraps
+from urllib.parse import quote_plus
+
+from flask import abort
+from flask import current_app
+from flask import redirect
+from flask import request
+from fsd_utils.authentication.utils import validate_token_rs256
+from jwt import ExpiredSignatureError
+from jwt import PyJWTError
+
+from .config import config_var_auth_host
+from .config import config_var_user_token_cookie_name
+
+
+def _failed_redirect(message: str = "Link expired or invalid"):
+    authenticator_host = current_app.config.get(config_var_auth_host)
+
+    if not authenticator_host:
+        current_app.logger.critical(
+            f"Failed Redirect: {config_var_auth_host} " "not set in environ"
```

Comment:
```suggestion
            f"Failed Redirect: {config_var_auth_host} not set in environ"
```
Don't need these extra quote marks since we're already using an f string.